sections/shorting/ShortingCard/ShortingCard.tsx

```diff
@@ -11,24 +11,46 @@ import useShort from '../hooks/useShort';
 
 import { CurrencyCardsSelector, ExchangeCardsWithSelector } from 'styles/common';
 
+import { useTranslation, Trans } from 'react-i18next';
+import { useRecoilValue } from 'recoil';
+import { isL2State } from 'store/wallet';
+
+import { MessageContainer, Message, MessageButton } from '../common';
+import useNetworkSwitcher from 'hooks/useNetworkSwitcher';
+
 const ShortingCard: FC = () => {
 	const { quoteCurrencyCard, baseCurrencyCard, footerCard } = useShort({
 		defaultBaseCurrencyKey: Synths.sETH,
 		defaultQuoteCurrencyKey: Synths.sUSD,
 	});
 
+	const { t } = useTranslation();
+	const isL2 = useRecoilValue(isL2State);
+	const { switchToL1, switchToL2 } = useNetworkSwitcher();
```

Comment:
Is switchToL1 used?

Reply:
it is not.  I saw the warning, but don't know how to handle it properly.  We explicitly return two values, so just omitting one of them in the deconstruction throws an error on execution.  I could alias it to _ or {} if you want.  Or create an alternate switcher function that only returns L2, and not L1.  Or whatever else you want.  Your call.

Reply:
Edit, of course, now that I try omitting it again, it works fine.  Sigh.  

Recommitting now.